coalib/output/ConsoleInteraction.py

```diff
@@ -560,26 +571,35 @@ def choose_action(console_printer, actions):
     :return:                Return choice of action of user.
     """
     while True:
-        console_printer.print(format_lines('*0: ' +
-                                           'Do nothing'))
+        dict_actions = {'(O)pen file': 'O', '(A)pply patch': 'A',
```

Comment:
It's by default in `possible_actions` array. The 1st action will be do nothing every time